lib/getPaths.js

```diff
@@ -0,0 +1,144 @@
+const path = require('path');
+const { encodePath, decodePath } = require('./uri-util');
+
+module.exports = {};
+
+function contains(parentPath, childPath) {
+    const relPath = path.relative(parentPath, childPath);
+    return (!(relPath.split(path.sep)[0] == '..' || path.isAbsolute(relPath)));
+}
+
```

Comment:
Can we add a jsdoc comment before this function explaining what it does and what the arguments/return values are?  I think we are losing some contextual information when we move this out into another file.

Reply:
Done: ff93240